backend/wargamebackend/wargamelogic/gamelogic/attack.py

```
@@ -0,0 +1,27 @@
+import math
+from .objects import *
+
+
+# RANGE_2_TILES_MULTIPLIER = .25
+
+def calculate_distance(tile_a, tile_b):
+    return math.ceil(math.sqrt((tile_a[0]-tile_b[0])**2 + (tile_a[1] - tile_b[1])**2))
+
+def calculate_attack_damage(attacker: GameUnit, target: GameUnit, attack: GameAttack):
```

Ruff (0.12.2)

Unused function argument: `attacker`

(ARG001)